plugins/terraform/src/api.test.ts

```diff
@@ -77,3 +77,74 @@ describe('TerraformApiClient', () => {
     );
   });
 });
+
+describe('TerraformApiClient - latest run', () => {
```

Comment:
Could this be incorporated in the existing describe block rather than a separate one for the new method added?

Reply:
I separated it as the single "terraform" describe block would get two large and unwieldy. 

Is there any disadvantage to having as separate describe block?

I can rename the first terraform describe block to "terraform component"

Reply:
Also, these tests need their own version of a single mockLatestRun object which the terraform tests do not need.  Similarly, these latest run tests need to resolve to this mock in the fetchApiMock.